src/app/pages/login/login.page.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { Router } from '@angular/router';
+import { render, RenderResult } from '@testing-library/angular';
+import { routerMock } from './../../shared/testUtils/router';
+import { LoginModule } from './login.module';
+import { LoginComponent } from './login.page';
+
+describe('LoginComponent', () => {
+  let component: RenderResult<LoginComponent, LoginComponent>;
+
+  beforeEach(async () => {
+    component = await render(LoginComponent, {
+      imports: [LoginModule],
+      providers: [
+        {
+          provide: Router,
+          useValue: routerMock,
+        },
+      ],
+    });
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
```

Comment:
Well tests could be better, couldn't they? :)

Reply:
Yes, I totally agree! I wanted to create a PR by the end of Sunday and skipped some tests. I will provide more testing in the next PR.